holoclean.py

```diff
@@ -104,7 +104,12 @@
         {'default': False,
          'dest': 'bias',
          'action': 'store_true',
-         'help': 'Use bias term'})
+         'help': 'Use bias term'}),
+    (tuple(['--printfw']),
+        {'default': False,
+         'dest': 'print_feat_weights',
```

Comment:
Could we change `print_feat_weights`  to `printfw` similar to all the other arguments that have the same dest name as the flag name?

Reply:
changed both to printfw